app/views/static/index.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>Static#index</h1>
```

Comment:
- [ ] After fixing the installation issues, your app does not work. When I visit the root path on the browser no greeting is displayed. You should have a `views/root/index.html.erb` file with `<div id="root">` in it. This will serve as the outlet for the react and redux part of the application. Kindly look into your app and ensure everything works.